products/slmicro5/profiles/stig.profile

```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
+
+metadata:
+    version: V1R1
+    SMEs:
+        - teacup-on-rockingchair
+
+reference: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux
+
+title: 'SUSE Linux Enterprise Micro (SLEM) 5'
```

Comment:
There should be STIG in the profile title

Reply:
Thanks added in [96606f8](https://github.com/ComplianceAsCode/content/pull/12028/commits/96606f82dc34b957a718811160de0abed1e87514)